src/utils/__fixtures__/test-inputs.ts

```diff
@@ -49,6 +49,13 @@ export const privateKeyUnlockedState = {
     transactions: [],
     transactionsError: undefined,
     loading: false,
+    pendingTransactions: {
+      local: [],
+      testnet: [],
+      mainnet: [],
+    },
+    // TODO: Would expect this nonce to be 0?
+    nonce: '1',
```

Comment:
@lukaw3d some unexpected behaviour occurred here, due to migration. I would expect this nonce to be '0', as it is probably missing in original state. (I added nonce to the Account interface in this PR) 

Also bellow, I would expect wallet to not include nonce at all - as it breaks the interface.

Reply:
Ah I see now that `nonce = 1` comes from mocked route here `**/chain/account/info/*`